### ISSUE TITLE
[ENH] Add BabyAFQ

### ISSUE THREAD

**Issue by mattcieslak** [PennLINC/qsiprep#767]
I hope to add a BabyAFQ recon workflow. @arokem and @36000 I have a question:

Does the current AFQ workflow download the bundle ROIs on the fly? If it does, would it make sense to add them to the qsiprep docker image? I will do the same with the BabyAFQ regions, whichever you prefer.

**Comment by 36000** [comment]
Yes, you are correct, I wrote something for the CLI that handles this. It would make sense to add a call to pyAFQ to the docker image like this:
```
pyAFQ download
```
That causes the CLI to call this function which will download all the templates for babyAFQ and normal AFQ: https://github.com/yeatmanlab/pyAFQ/blob/899d504c7b75cca7a41f3c48daed9c614754d762/bin/pyAFQ#L117

**Comment by mattcieslak** [comment]
@36000 `pyAFQ download` is amazing. Did that get the baby atlases too?

**Comment by 36000** [comment]
Yup! It gets the baby atlases. I wrote it for my own docker images, this is its exact intended use.

**Comment by mattcieslak** [comment]
@36000 I think something may have changed in pyAFQ such that the config from qsiprep doesn't work anymore. I added `pyAFQ download` to the DockerFile and there is a `/home/qsiprep/AFQ_data` that [has the ROIs](https://app.circleci.com/pipelines/github/PennLINC/qsiprep_build/236/workflows/a1c3382b-133a-4f42-9b8a-8ae0c3769173/jobs/2253?invite=true#step-103-292565_106). 

Here the error always ends with (TypeError: bundle_info must be a dict, or a BundleDict)[https://app.circleci.com/pipelines/github/PennLINC/qsiprep/1887/workflows/8fe46ea2-e7ca-4086-81ee-66ab7234f91f/jobs/21871?invite=true#step-104-65124_54]. I tried changing the config file [here](https://github.com/PennLINC/qsiprep/pull/768/files#diff-59fc53e4bd0e05baa08a37c92a71f7d9c097f66efb6560cdc6b6d20ac218d44dL86) but there is still a problematic bundle_info somewhere. Any ideas?

**Comment by mattcieslak** [comment]
aha, it looks like in the CI tests when using the `--sloppy` flag `bundle_info` gets set to `'bundle_info': ['SLF_L', 'ARC_L', 'CST_L', 'CST_R'],` which is indeed not a dict. What is this arg supposed to look like if I only want to test for a couple bundles?

**Comment by 36000** [comment]
Ah! Them being a list is actually fine as the software should convert them to a dict. The problem is we got rid of the acronyms. There is a way to calculate an updated JSON. You run `pyAFQ -q`. For this list, you would need to update it to:
```
['Left Superior Longitudinal', 'Left Arcuate', 'Left Corticospinal', 'Right Corticospinal']
```
I went ahead and generated some updated JSON's off the pyAFQ master.
[pyafq_input_trk.json](https://github.com/user-attachments/files/16072800/pyafq_input_trk.json)
[pyafq.json](https://github.com/user-attachments/files/16072801/pyafq.json)

**Comment by 36000** [comment]
Oh - actually - i lied. You do need to instantiate a bundledict and pass it to pyAFQ if you want a subset. Please hold while I find the syntax.

**Comment by 36000** [comment]
The line should be:
```
"bundle_info": 'default18_bd()["Left Superior Longitudinal", "Left Arcuate", "Left Corticospinal", "Right Corticospinal"]'
```

**Comment by mattcieslak** [comment]
That did it! Thank you @36000

**Comment by mattcieslak** [comment]
One more question - do I need to install pyafq in a different way in order to be able to make the plots? I'm installing pyafq [here](https://github.com/PennLINC/qsiprep_build/blob/main/qsiprep_env.yml#L26)

**Comment by 36000** [comment]
which plots? For the plotly html files we generate, this will work. There are other plots that require optional installations.
